### DISCUSSION TITLE
SSL error when activating/deactivating php extension

### DISCUSSION THREAD

**Discussion by luisAntonioLAGS** [leokhoa/laragon#747] [General]
When I activate/deactivate a PHP extension from the graphical interface, SSL is deactivated and in my case Firefox does not load the project, displaying a message of a potential security error.

![imagen](https://github.com/leokhoa/laragon/assets/60680854/2845f7aa-3c8d-401d-95e3-8b156978cfbb)
![imagen](https://github.com/leokhoa/laragon/assets/60680854/93689b00-06b0-4499-bf63-2750eca75014)

I hope this bug is resolved in the new version 🙏.

**Comment by Pen-y-Fan** [comment]
See my blog [How to set up Laragon on a new Windows computer (part 4) - Enable security](https://pen-y-fan.github.io/2022/11/10/how-to-set-up-laragon-on-a-new-windows-computer-part-4/), which may help.

**Reply by luisAntonioLAGS** [reply]
Thanks :)
